tools/shared.py

```diff
@@ -1089,16 +1089,6 @@ def include_directive(paths):
 except NameError:
   pass
 
-# If we have 'env', we should use that to find python, because |python| may fail while |env python| may work
```

Comment:
Should we not do this anymore? Why is this safe to remove?

Reply:
ENV_PREFIX is not used anywhere, I just noticed that while making this change.  I can separate if you prefer.